flow/model/index.js

```diff
@@ -0,0 +1,60 @@
+// @flow
+
+declare module 'mattermost-typed/model' {
+    declare export type User = mattermost$User
```

Comment:
User and some other types are going to be any. Maybe declarations have some conflict with `export` :(

Reply:
resolved